dataland-e2etests/src/test/kotlin/org/dataland/e2etests/utils/ApiAccessor.kt

```diff
@@ -45,6 +45,10 @@ class ApiAccessor {
 
     val jwtHelper = JwtAuthenticationHelper()
 
+    fun authenticateAsTechnicalUser(technicalUser: TechnicalUser) {
```

Comment:
Why do we need this method?
In other places in ApiAcesssor we just useauthenticateApiCallsWithJwtForTechnicalUser(technicalUser) directly.
This should be consistent

Reply:
Seems to be gone now, closing this